packages/common/src/services/audius-backend/solana.ts

```diff
@@ -23,6 +23,7 @@ import { BN_USDC_CENT_WEI } from '~/utils/wallet'
 import { AnalyticsEvent, ID, Name, SolanaWalletAddress } from '../../models'
 
 import { AudiusBackend } from './AudiusBackend'
+import { PurchaseAccess } from '~/store'
```

Comment:
nit: It feels odd to import from the store in audius-backend? I think maybe this enum should be in models.

Reply:
good call